mergify_engine/context.py

```diff
@@ -1066,6 +1066,15 @@ class PullRequest:
         "files",
     }
 
+    BACKPORT = "backport"
+    COPY = "copy"
+
+    BRANCH_PREFIX_MAP = {BACKPORT: "bp", COPY: "copy"}
+
+    @staticmethod
+    def branch_prefix(kind):
+        return PullRequest.BRANCH_PREFIX_MAP[kind]
```

Comment:
This is really related to a couple of actions so that should be part of the concerned `Action` classes and not leaked into the `Context`.

Reply:
I was initially tempted but somehow I wrongly thought the PullRequest class was the one. Thanks for the clarification :)